primitives/millau/src/lib.rs

```diff
@@ -68,23 +71,25 @@ impl sp_runtime::traits::Hash for BlakeTwoAndKeccak256 {
 	}
 }
 
-/// Maximal weight of single Millau block.
-pub const MAXIMUM_BLOCK_WEIGHT: Weight = 10_000_000_000;
-/// Portion of block reserved for regular transactions.
-pub const AVAILABLE_BLOCK_RATIO: u32 = 75;
-/// Maximal weight of single Millau extrinsic (65% of maximum block weight = 75% for regular
-/// transactions minus 10% for initialization).
-pub const MAXIMUM_EXTRINSIC_WEIGHT: Weight = MAXIMUM_BLOCK_WEIGHT / 100 * (AVAILABLE_BLOCK_RATIO as Weight - 10);
-/// Maximal size of Millau block.
-pub const MAXIMUM_BLOCK_SIZE: u32 = 2 * 1024 * 1024;
-/// Maximal size of single normal Millau extrinsic (75% of maximal block size).
-pub const MAXIMUM_EXTRINSIC_SIZE: u32 = MAXIMUM_BLOCK_SIZE / 100 * AVAILABLE_BLOCK_RATIO;
+/// Maximum weight of single Millau block.
+///
+/// This represents two seconds of compute assuming a target block time of six seconds.
+pub const MAXIMUM_BLOCK_WEIGHT: Weight = 2 * WEIGHT_PER_SECOND;
```

Comment:
The value is now the same as in Rialto. I'd prefer to keep different weight/size limits. The idea was to:
1) change `Millau` to chain that operates differently from `Rialto` (i.e. it has different crypto/hash algorithms, block times, consensus algorithms). This is WIP, but every time I introduce/change some constants, they're a bit different from `Millau`. The direction is to make `Millau` lighter+faster than `Rialto`;
2) that isn't something I work on (and probably there are better ways to do that), but these constant in different chains may be used to compare messages bandwidth under different limits.

So unless you have any arguments against keeping previous constants (or at least having different values in different chains), let's change that.